### PR TITLE
example_test.py: fixup coinbase height argument, derive number clearly

### PR DESCRIPTION
**Pull request by instagibbs**

### CODE REVIEW

**Comment by MarcoFalke** [comment]
utACK 6c5355e43d7c68c9ba71bc3170b15a6cf96bb9b4

**Comment by promag** [comment]
utACK 6c5355e.

File: test/functional/example_test.py [hunk 1]
```diff
@@ -164,13 +164,13 @@ def run_test(self):
         self.tip = int(self.nodes[0].getbestblockhash(), 16)
         self.block_time = self.nodes[0].getblock(self.nodes[0].getbestblockhash())['time'] + 1
 
-        height = 1
+        height = self.nodes[0].getblockcount()
```
Comment (MarcoFalke) [inline test/functional/example_test.py]: Actually the ` + 1` should be suffixed here (similar to block_time above?)

And then remove the change further down?

```suggestion
        height = self.nodes[0].getblockcount() + 1
```

Comment (instagibbs) [reply test/functional/example_test.py]: 2+10=12, and we're waiting for block 11, I'd have to change that too.

Comment (MarcoFalke) [reply test/functional/example_test.py]: Ok, fine.

**Pull request merged**